document/src/vespa/document/fieldset/fieldsetrepo.cpp

```diff
@@ -124,7 +128,13 @@ FieldSetRepo::configureDocumentType(const DocumentType & documentType) {
     for (const auto & entry : documentType.getFieldSets()) {
         vespalib::string fieldSetName(documentType.getName());
         fieldSetName.append(':').append(entry.first);
-        _configuredFieldSets[fieldSetName] = parse(_doumentTyperepo, fieldSetName);
+        try {
+            auto fieldset = parse(_doumentTyperepo, fieldSetName);
+            _configuredFieldSets[fieldSetName] = std::move(fieldset);
+        } catch (const FieldNotFoundException & ex) {
```

Comment:
It is probably better to just ignore these silently (since they are legal when used in context of the proton index schema) and rather warn about usage later if needed.